src/Toolkit/Collection.php

```diff
@@ -1055,6 +1055,30 @@ public function values(): array
         return array_values($this->data);
     }
 
+    /**
+     * The when method only executes the given Closure when the first parameter
+     * is true. If the first parameter is false, the Closure will not be executed.
+     * You may pass another Closure as the third parameter to the when method.
+     * This Closure will execute if the first parameter evaluates as false
+     *
+     * @param mixed $condition
+     * @param Closure $callback
+     * @param Closure $fallback
+     * @return Collection
```

Comment:
This should be `@return mixed` as the return value depends on the return value of the Closure.

Reply:
You right, any method in the collection can be used 👍